python/cugraph-pyg/pytest.ini

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[pytest]
+addopts =
+           --benchmark-warmup=off
+           --benchmark-max-time=0
+           --benchmark-min-rounds=1
+           --benchmark-columns="mean, rounds"
+           ## do not run the slow tests/benchmarks by default
+           -m "not slow"
+           ## for use with rapids-pytest-benchmark plugin
+           #--benchmark-gpu-disable
+           ## for use with pytest-cov plugin
+           #--cov=cugraph
+           #--cov-report term-missing:skip-covered
+
+markers =
+          managedmem_on: RMM managed memory enabled
+          managedmem_off: RMM managed memory disabled
+          poolallocator_on: RMM pool allocator enabled
+          poolallocator_off: RMM pool allocator disabled
+          preset_gpu_count: Use a hard-coded number of GPUs for specific MG tests
+          ETL: benchmarks for ETL steps
+          small: small datasets
+          tiny: tiny datasets
+          directed: directed datasets
+          undirected: undirected datasets
+          cugraph_types: use cuGraph input types
+          nx_types: use NetworkX input types
+          matrix_types: use SciPy/CuPy matrix input types
+          slow: slow-running tests/benchmarks
```

Comment:
Are we using any of these `markers` in the pytests ?

Reply:
no, I'm going to take these out